README.md

```diff
@@ -1,15 +1,16 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Awesome Books
-In this exercise I  build a simple yet powerful webpack boilerplate, which I can later use as a starting point in my projects. 
+# To-Do-List
+In this project, I build a simple HTML list of To Do tasks. The list will be styled according to the specifications. This simple web page is built using webpack and served by a webpack dev server.
 
  ## Learning objectives
 - Use webpack to bundle JavaScript.
+- Learn how to use proper ES6 syntax.
+- Use ES6 modules to write modular JavaScript.
 
 ## Built With
 
 - HTML, CSS, JavaScript
```

Comment:
- [ ] I noticed that the link to the license is broken :cry:  Please add a Licence to your repo in order to make it more professional :+1:  We recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.

- [ ] Please add a link to issues for contributing to Contributing header section, this will help people to contribute or do some requests to your project. Please fix it :smile: